openml/tasks/task.py

```diff
@@ -109,32 +121,39 @@ def remove_tag(self, tag: str):
         data = {'task_id': self.task_id, 'tag': tag}
         openml._api_calls._perform_api_call("/task/untag", 'post', data=data)
 
-    def _to_dict(self) -> Dict[str, OrderedDict]:
+    def _to_dict(self) -> dict:
```

Comment:
Why change this? Anyway, it should be `Dict` instead of `dict`. In case you changed it because of the ordered dict, you can use `'OrderedDict[str, OrderedDict]'`.

Reply:
I wanted to change it to `OrderedDict`, however I decided to check the notation used in flows and to be consistent I used the same one. Both functions return an `OrderedDict.`
https://github.com/openml/openml-python/blob/c7db12287aa25415c854d15549cfaa05835cd7d6/openml/flows/flow.py#L151

Personally I would want `OrderedDict[str, OrderedDict]`.

Reply:
Then go for it.

Reply:
I already did it, I did not resolve the conversation so it would act as a reminder, in case you wanted to edit the flows `_to_dict`.

Reply:
Not now, this is about task upload.